src/main/java/com/zsmartsystems/zigbee/sniffer/ZigBeeSniffer.java

```diff
@@ -396,6 +419,15 @@ public synchronized void emberMfgLibPacketReceived(int lqi, int rssi, int[] data
             return false;
         }
 
+        lastChannelRotationTimestamp = System.currentTimeMillis();
+
         return true;
     }
+
+    private static ZigBeeChannel getNextChannel() {
+        if (channelId == ZigBeeChannel.CHANNEL_26.getChannel()) {
```

Comment:
Note that this will cycle through channels that do not exist in most devices. 2.4GHz channels (as supported on most Ember devices) are channels 11 to 26, and possibly setting to a channel below 11 may cause problems (although I've not tried this). Even if there is no error, this is still causing the scan to spend nearly 50% of the time on channels that do not exist).

Reply:
I added two command line option to specify the channel range to scan. If not specified, the sniffer will scan by default range 11-26.